bot/exts/fun/latex.py

```diff
@@ -105,6 +115,16 @@ async def _upload_to_pastebin(self, text: str) -> str | None:
         except web.HTTPClientError as e:
             log.info("Error when uploading latex output to pastebin. %s", e)
 
+    async def _prepare_error_embed(self, title: str, err: Exception | None) -> discord.Embed:
+        embed = discord.Embed(title=title)
+        embed.description = "No logs available."
+        if err is not None and err.logs is not None:
```

Comment:
You could simplify this to
```suggestion
        if hasattr(err, "logs")
```

Reply:
Thanks for the suggestion :), I gave it a try, yet this will only check if the attr exist and not its nullability
So now I use getattr (which is used internally by hasattr, https://docs.python.org/3/library/functions.html#hasattr)